src/style.css

```diff
@@ -0,0 +1,134 @@
+@font-face {
+  font-family: Avenir_heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: avenir_book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
```

Comment:
do not use `*` for all styles

leave only `box-sizing: border-box;` here

move  `font-family` to `html` selector and margin, padding to where you need to use it